samples/NativeCatalog/ios/Podfile

```diff
@@ -35,8 +35,8 @@ target 'NativeCatalog' do
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
   
   pod 'react-native-pspdfkit', :path => '../node_modules/react-native-pspdfkit'  
-  pod 'PSPDFKit', podspec:'https://customers.pspdfkit.com/cocoapods/YOUR_COCOAPODS_KEY_GOES_HERE/pspdfkit/latest.podspec'
-  pod 'Instant', podspec:'https://customers.pspdfkit.com/cocoapods/YOUR_COCOAPODS_KEY_GOES_HERE/instant/latest.podspec'
+  pod 'PSPDFKit', podspec:'https://customers.pspdfkit.com/pspdfkit-ios/latest.podspec'
+  pod 'Instant', podspec:'https://customers.pspdfkit.com/instant/latest.podspec'
```

Comment:
These links do not currently work because 9.4 is not yet release. For testing, please use the staging environment instead.